v1.1/select.md

```diff
@@ -550,6 +569,26 @@ OFFSET 5;
 +----+------------------+
 ~~~
 
+Alternatively, you can use `FETCH NEXT`.
```

Comment:
Same as above. Let's remove this alternate example and just go with the callout in the parameters table.